flows/classifier_specs/staging.yaml

```diff
@@ -74,3 +46,31 @@
 - Q973:v5
 - Q983:v5
 - Q986:v6
+- Q1016:v0
```

Comment:
question: Do we need to do sandbox.yaml and labs.yaml as well?

Reply:
There were no changes to commit for those environments